Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/model/DocumentMetadataResult.java

```diff
@@ -100,11 +100,81 @@
     private String authorInstitution;
 
     /**
+     * The author Role.
+     */
+    private String authorRole;
+
+    /**
+     * The author Specialty.
+     */
+    private String authorSpecialty;
+    
+    /**
      * The event codes.
      */
     private String eventCodes;
 
     /**
+     * The Coding Scheme.
+     */
+    private String codingScheme;
+    
+    /**
+     * The Coding Scheme Name.
+     */
+    private String codingSchemeName;
+    
+    /**
+     * The Hash.
+     */
+    private String hash;
+    
+    /**
+     * The intendedRecipient.
+     */
+    private String intendedRecipient;
+
+    /**
+     * The Language Code.
+     */
+    private String languageCode;
+
+    /**
+     * The Legal Authenticator.
+     */
+    private String legalAuthenticator;
+
+    /**
+     * The name.
+     */
+    private String name;
+
+    private String description;
+
+    /**
+     * The mimeType.
+     */
+    private String mimeType;
+    
+    private boolean Opague;
+    
+    private int Size;
+    
+    private String sourcePatientId;
+    
+    private String URI;
```

Comment:
For member vars that are acronyms, we typically use all lowercase names (e.g., "hcid", "uri").

For this comment, and the ones above, make sure the getter/setter method names reflect the changes.